mlflow/pipelines/steps/ingest/__init__.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, step_config: Dict[str, Any], pipeline_root: str):
         super().__init__(step_config, pipeline_root)
 
         dataset_format = step_config.get("format")
+        self.disable_profiling = step_config.get("disable_profiling", False)
```

Comment:
`disable_profiling` might be a little confusing what it means exactly. How about something more specific? e.g. `skip_data_profiling`

Reply:
Good point